### PR TITLE
Test build in travis

### LINKED ISSUE #125: Add build check on Travis
Travis is now only running lint and test.
It can happen (it happened) that the build will fail in drone after the PR is merged -> what could be a big problem (edited)
We should add to the Travis rules the building of the app, so we'll notice the app is not buildable in advance (before merging)

### PR DESCRIPTION
**Pull request by dpordomingo**
fix #125 

once this is merged, Travis will check if project is buildable

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,10 +9,13 @@ before_install:
...
+jobs:
+  include:
+    - stage: Tests, linters and build
+      script:
+        - make lint-frontend
+        - make test-frontend
+        - make lint-backend
+        - make test
+    - script:
+        - make build-app
```
Comment (smacker) [inline .travis.yml]: no need to create tars for testing.

Comment (dpordomingo) [reply .travis.yml]: The issue requires to test the whole building process

Comment (smacker) [reply .travis.yml]: creating tars isn't building.

Comment (dpordomingo) [reply .travis.yml]: Just called `make packages` and whatever it does, is ok for the purpose of the issue.
Is there any problem if the tars are generated?

Comment (smacker) [reply .travis.yml]: nothing really wrong. just my perfectionism 😅 
Task has the name `test build`. Comment in travis says `build`. And we have a target `make build` exactly for this purpose - to build.

Comment (dpordomingo) [reply .travis.yml]: Where is the name `test build`?

aside:
`build` rule is called from `packages` rule, that is called from `build-app` that is what drone really calls, that is what we're trying to reproduce.

Comment (smacker) [reply .travis.yml]: I meant the name of the job:`stage: Tests, linters and build`.

As you may notice I didn't request changes, it's okay to merge it as it is.

**Comment by bzz** [comment]
@dpordomingo rebase seems to be in order.

Also I'm confused - review requested, but #125 is not in "Code Review" state according to the Kanban.

**Review by bzz — APPROVED** [review]
LGTM, sans https://github.com/src-d/code-annotation/pull/141#issuecomment-367266375

**Pull request merged**